libs/common/src/types/tfm/cancellation-status.ts

```diff
@@ -0,0 +1,4 @@
+import { TFM_DEAL_CANCELLATION_STATUS } from '../../constants';
+import { ValuesOf } from '..';
+
+export type TfmDealCancellationStatus = ValuesOf<typeof TFM_DEAL_CANCELLATION_STATUS>;
```

Comment:
I probably should have done this too oops! Could you use this type in `TfmDealCancellationWithStatus` please :)

Reply:
Yes good idea! Have changed, thanks